website/docs/guides/version-2-upgrade.html.md

```diff
@@ -644,6 +646,31 @@ Previously, importing this resource resulted in an `aws_route` resource for each
 addition to the `aws_route_table`, in the Terraform state. Support for importing `aws_route` resources has been added and importing this resource only adds the `aws_route_table` 
 resource, with in-line routes, to the state.
 
+## Resource: aws_route53_record
+
+### allow_overwrite Default Value Change
+
+The resource now requires existing Route 53 Records to be imported into the Terraform state for management unless the `allow_overwrite` argument is enabled. The `allow_overwrite` flag is considered deprecated for removal in the next major version of the Terraform AWS Provider (version 3.0.0).
```

Comment:
Suggestion: don't name a specific version, just say "a future version". Then if you forget for 3.0.0, you didn't break a promise.

Reply:
👍 Great suggestion